os/provisioning.md

```diff
@@ -32,9 +32,9 @@ etcd:
   discovery:                   https://discovery.etcd.io/<token>
 ```
 
-`PUBLIC_IPV4` and `PRIVATE_IPV4` are automatically populated from the environment (such as EC2, Azure, or GCE) in which Container Linux runs, if this metadata exists. For example: For EC2 (the default), the `public_ipv4` and `local_ipv4` metadata would be used. In Azure, *either* the virtual IP or public IP could be used for the `PUBLIC_IPV4` and the dynamic IP would be used for the `PRIVATE_IPV4`. On bare metal, this information cannot be reliably derived in a general manner, so these variables cannot be used.
+`PUBLIC_IPV4` and `PRIVATE_IPV4` are automatically populated from the environment in which Container Linux runs, if this metadata exists. Given the many different environments in which Container Linux can run, it's difficult if not impossible to accurately determine these variables in every instance. Be certain to check this value as a troubleshooting measure.
 
-Given the many different environments in which Container Linux can run, it's difficult if not impossible to accurately determine these variables in every instance.
+For example, the default metadata for an EC2 environment would be used: `public_ipv4` and `local_ipv4`. On Azure, *either* the virtual IP or public IP could be used for the `PUBLIC_IPV4` and the dynamic IP would be used for the `PRIVATE_IPV4`. On bare metal, this information cannot be reliably derived in a general manner, so these variables cannot be used.
```

Comment:
On Azure, *either* the virtual IP or public IP could be used for the `PUBLIC_IPV4` (`ct` makes a best guess and uses the virtual IP, but this could change in the future) and the dynamic...